workflows/test/transaction-monitor-workflow.test.js

```diff
@@ -0,0 +1,341 @@
+/** @import {WorkflowEvent, WorkflowStep} from 'cloudflare:workers' */
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest'
+import { TransactionMonitorWorkflow } from '../lib/transaction-monitor-workflow.js'
+
+vi.mock('../lib/chain.js', () => ({
+  getChainClient: vi.fn(),
+}))
+
+describe('TransactionMonitorWorkflow', () => {
+  let workflow
+  let mockEnv
+  let mockStep
+  let mockQueue
```

Comment:
These variables have type `any`, which disables type checks. Let's tell TypeScript about their types.


```suggestion
  /** @type {TransactionMonitorWorkflow} */
  let workflow
  /**
   * @type {{
   *   TRANSACTION_QUEUE: {
   *     send: Mock<Procedure>
   *   }
   *   ENVIRONMENT: string
   *   RPC_URL: string
   *   FILCDN_CONTROLLER_ADDRESS_PRIVATE_KEY: string
   * }}
   */
  let mockEnv
  /**
   * @type {{
   *   do: Mock<Procedure>
   * }}
   */
  let mockStep
  /**
   * @type {{
   *   send: Mock<Procedure>
   * }}
   */  
  let mockQueue
```

Reply:
Added in https://github.com/filbeam/worker/pull/357/commits/e46a0a728d06ba5ac65aceab099c36396a94c50d